content/docs/command-reference/data/index.md

```diff
@@ -0,0 +1,24 @@
+# data
+
+Contains a command that shows changes in data tracked by dvc:
+[status](/doc/command-reference/data/status).
+
+## Synopsis
+
+```usage
+usage: dvc data [-h] [-q | -v] {status} ...
+
+positional arguments:
+  COMMAND
+    status         Show changes between the last git commit, the dvcfiles and the workspace.
```

Comment:
```suggestion
    status         Show changes between the last Git commit,
                   DVC metafiles, and the workspace.
```

Reply:
The `metafile` is only docs-concept, so I'll prefer not to introduce it in CLI.

Reply:
but `dvcfiles` is also a weird term ... we should be consistent across docs and DVC itself.

Reply:
@shcheklein, this term has not been introduced in DVC (maybe for good reasons).

Reply:
it's an internal one though? so, we should be following the same logic and not introduce internal terms into help / docs. Rather we should think what actually fits better / easier to read and understand